tldw_Server_API/app/main.py

```
@@ -797,7 +797,10 @@ def _dict_config_wrapper(config):
             _dict_config_wrapper._tldw_original = getattr(_logcfg, "_tldw_original_dictConfig", None)
             _logcfg._tldw_dict_config_wrapped = True  # type: ignore[attr-defined]
 except _LOGGING_SETUP_EXCEPTIONS as _log_wrap_err:
-    logger.debug(f"Failed to wrap logging.config.dictConfig for interception: {_log_wrap_err}")
+    logger.debug(
```

Ruff (0.15.6)

[error] Too many arguments for `logging` format string

(PLE1205)